tasks/main.yml

```diff
@@ -45,4 +47,4 @@
 
 - name: Delete temp directory
   file: path={{ ffmpeg_temp_path }}
-        state=absent
```

Comment:
i edited the file on github, did not intended to remove that line.
 apologies.

On Fri, Apr 22, 2016 at 5:19 AM, hedii <notifications@github.com> wrote:

> In tasks/main.yml
> <https://github.com/hedii/ansible-role-ffmpeg/pull/1#discussion_r60711985>
> :
>
> > @@ -45,4 +47,4 @@
> >
> >  - name: Delete temp directory
> >    file: path={{ ffmpeg_temp_path }}
> > -        state=absent
>
> i don't get this one. this block will do nothing if state=absent is removed
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hedii/ansible-role-ffmpeg/pull/1/files/7f9477af420f0902d14115ef18239b4809e51964#r60711985>
>